setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import find_packages, setup
 
-install_requires = ["numpy", "sklearn", "pandas", "scipy", "scikit-learn"]
+install_requires = ["numpy", "sklearn", "pandas==1.3.5", "scipy", "scikit-learn"]
```

Comment:
I'd suggest <= 1.3.5 since this probably is a bit too restrictive at the moment

I hope the pandas new release things gets cleared up asap

Reply:
Thanks, changed to 1.3.4 now :)